Lets Do This/Classes/AppDelegate.m

```diff
@@ -38,6 +39,11 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     [application registerUserNotificationSettings:settings];
     [application registerForRemoteNotifications];
 
+    LDTUserConnectViewController *rootVC = [[LDTUserConnectViewController alloc] initWithNibName:@"LDTUserConnectView" bundle:nil];
```

Comment:
@eroth Does this look correct? Every time you load a View Controller with a nib -- its this way?